01-18 16:00:19 jdbc[2]: SQLException
org.h2.jdbc.JdbcSQLException: Table LINKRULES not found; SQL statement:
DELETE FROM  LINKRULES; [42S02-60]
	at org.h2.message.Message.getSQLException(Message.java:84)
	at org.h2.message.Message.getSQLException(Message.java:88)
	at org.h2.message.Message.getSQLException(Message.java:66)
	at org.h2.command.Parser.readTableOrView(Parser.java:3911)
	at org.h2.command.Parser.readTableOrView(Parser.java:3891)
	at org.h2.command.Parser.readSimpleTableFilter(Parser.java:645)
	at org.h2.command.Parser.parseDelete(Parser.java:663)
	at org.h2.command.Parser.parsePrepared(Parser.java:304)
	at org.h2.command.Parser.parse(Parser.java:260)
	at org.h2.command.Parser.parse(Parser.java:232)
	at org.h2.command.Parser.prepareCommand(Parser.java:204)
	at org.h2.engine.Session.prepareLocal(Session.java:172)
	at org.h2.engine.Session.prepareCommand(Session.java:154)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:971)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:1208)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:165)
	at com.zaxxer.hikari.pool.ProxyConnection.prepareStatement(ProxyConnection.java:311)
	at com.zaxxer.hikari.pool.HikariProxyConnection.prepareStatement(HikariProxyConnection.java)
	at helio.materialiser.evaluator.H2Evaluator.closeH2Cache(H2Evaluator.java:432)
	at helio.materialiser.evaluator.H2Evaluator.initH2Cache(H2Evaluator.java:123)
	at helio.materialiser.HelioMaterialiser.<init>(HelioMaterialiser.java:43)
	at helio.materialiser.test.utils.TestUtils.generateRDFSynchronously(TestUtils.java:56)
	at helio.materialiser.issues.GithubIssuesTest.testIssue7(GithubIssuesTest.java:27)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.internal.runners.TestMethod.invoke(TestMethod.java:59)
	at org.junit.internal.runners.MethodRoadie.runTestMethod(MethodRoadie.java:98)
	at org.junit.internal.runners.MethodRoadie$2.run(MethodRoadie.java:79)
	at org.junit.internal.runners.MethodRoadie.runBeforesThenTestThenAfters(MethodRoadie.java:87)
	at org.junit.internal.runners.MethodRoadie.runTest(MethodRoadie.java:77)
	at org.junit.internal.runners.MethodRoadie.run(MethodRoadie.java:42)
	at org.junit.internal.runners.JUnit4ClassRunner.invokeTestMethod(JUnit4ClassRunner.java:88)
	at org.junit.internal.runners.JUnit4ClassRunner.runMethods(JUnit4ClassRunner.java:51)
	at org.junit.internal.runners.JUnit4ClassRunner$1.run(JUnit4ClassRunner.java:44)
	at org.junit.internal.runners.ClassRoadie.runUnprotected(ClassRoadie.java:27)
	at org.junit.internal.runners.ClassRoadie.runProtected(ClassRoadie.java:37)
	at org.junit.internal.runners.JUnit4ClassRunner.run(JUnit4ClassRunner.java:42)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
01-19 11:52:28 database: opening /home/salva/git/helio-materialiser/helio-storage/helio-engine
org.h2.jdbc.JdbcSQLException: Database may be already in use: Locked by another process. Possible solutions: close all other connection(s); use the server mode [90020-60]
	at org.h2.message.Message.getSQLException(Message.java:84)
	at org.h2.message.Message.getSQLException(Message.java:88)
	at org.h2.message.Message.getSQLException(Message.java:66)
	at org.h2.store.FileLock.error(FileLock.java:310)
	at org.h2.store.FileLock.lockFile(FileLock.java:173)
	at org.h2.store.FileLock.lock(FileLock.java:69)
	at org.h2.engine.Database.open(Database.java:438)
	at org.h2.engine.Database.<init>(Database.java:186)
	at org.h2.engine.Engine.openSession(Engine.java:50)
	at org.h2.engine.Engine.getSession(Engine.java:98)
	at org.h2.engine.Session.createSession(Session.java:139)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:928)
	at org.h2.jdbcx.JdbcDataSource.getJdbcConnection(JdbcDataSource.java:109)
	at org.h2.jdbcx.JdbcDataSource.getConnection(JdbcDataSource.java:96)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:369)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:198)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:467)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:541)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:115)
	at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:81)
	at helio.materialiser.evaluator.H2Evaluator.createInitHikari(H2Evaluator.java:71)
	at helio.materialiser.evaluator.H2Evaluator.<init>(H2Evaluator.java:52)
	at helio.materialiser.configuration.HelioConfiguration.<clinit>(HelioConfiguration.java:97)
	at helio.materialiser.engine.DataTypesTest.createOneDataType(DataTypesTest.java:14)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.internal.runners.TestMethod.invoke(TestMethod.java:59)
	at org.junit.internal.runners.MethodRoadie.runTestMethod(MethodRoadie.java:98)
	at org.junit.internal.runners.MethodRoadie$2.run(MethodRoadie.java:79)
	at org.junit.internal.runners.MethodRoadie.runBeforesThenTestThenAfters(MethodRoadie.java:87)
	at org.junit.internal.runners.MethodRoadie.runTest(MethodRoadie.java:77)
	at org.junit.internal.runners.MethodRoadie.run(MethodRoadie.java:42)
	at org.junit.internal.runners.JUnit4ClassRunner.invokeTestMethod(JUnit4ClassRunner.java:88)
	at org.junit.internal.runners.JUnit4ClassRunner.runMethods(JUnit4ClassRunner.java:51)
	at org.junit.internal.runners.JUnit4ClassRunner$1.run(JUnit4ClassRunner.java:44)
	at org.junit.internal.runners.ClassRoadie.runUnprotected(ClassRoadie.java:27)
	at org.junit.internal.runners.ClassRoadie.runProtected(ClassRoadie.java:37)
	at org.junit.internal.runners.JUnit4ClassRunner.run(JUnit4ClassRunner.java:42)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:89)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:41)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:542)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:770)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:464)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)
01-19 12:06:40 database: opening /home/salva/git/helio-materialiser/helio-storage/helio-engine
org.h2.jdbc.JdbcSQLException: Database may be already in use: Locked by another process. Possible solutions: close all other connection(s); use the server mode [90020-60]
	at org.h2.message.Message.getSQLException(Message.java:84)
	at org.h2.message.Message.getSQLException(Message.java:88)
	at org.h2.message.Message.getSQLException(Message.java:66)
	at org.h2.store.FileLock.error(FileLock.java:310)
	at org.h2.store.FileLock.lockFile(FileLock.java:173)
	at org.h2.store.FileLock.lock(FileLock.java:69)
	at org.h2.engine.Database.open(Database.java:438)
	at org.h2.engine.Database.<init>(Database.java:186)
	at org.h2.engine.Engine.openSession(Engine.java:50)
	at org.h2.engine.Engine.getSession(Engine.java:98)
	at org.h2.engine.Session.createSession(Session.java:139)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:928)
	at org.h2.jdbcx.JdbcDataSource.getJdbcConnection(JdbcDataSource.java:109)
	at org.h2.jdbcx.JdbcDataSource.getConnection(JdbcDataSource.java:96)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:369)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:198)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:467)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:541)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:115)
	at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:81)
	at helio.materialiser.evaluator.H2Evaluator.createInitHikari(H2Evaluator.java:71)
	at helio.materialiser.evaluator.H2Evaluator.<init>(H2Evaluator.java:52)
	at helio.materialiser.configuration.HelioConfiguration.<clinit>(HelioConfiguration.java:97)
	at helio.materialiser.engine.DataTypesTest.createOneDataType(DataTypesTest.java:14)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.internal.runners.TestMethod.invoke(TestMethod.java:59)
	at org.junit.internal.runners.MethodRoadie.runTestMethod(MethodRoadie.java:98)
	at org.junit.internal.runners.MethodRoadie$2.run(MethodRoadie.java:79)
	at org.junit.internal.runners.MethodRoadie.runBeforesThenTestThenAfters(MethodRoadie.java:87)
	at org.junit.internal.runners.MethodRoadie.runTest(MethodRoadie.java:77)
	at org.junit.internal.runners.MethodRoadie.run(MethodRoadie.java:42)
	at org.junit.internal.runners.JUnit4ClassRunner.invokeTestMethod(JUnit4ClassRunner.java:88)
	at org.junit.internal.runners.JUnit4ClassRunner.runMethods(JUnit4ClassRunner.java:51)
	at org.junit.internal.runners.JUnit4ClassRunner$1.run(JUnit4ClassRunner.java:44)
	at org.junit.internal.runners.ClassRoadie.runUnprotected(ClassRoadie.java:27)
	at org.junit.internal.runners.ClassRoadie.runProtected(ClassRoadie.java:37)
	at org.junit.internal.runners.JUnit4ClassRunner.run(JUnit4ClassRunner.java:42)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:89)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:41)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:542)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:770)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:464)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)
